Gemfile

```diff
@@ -10,8 +10,8 @@ gem 'pkg-config', '~> 1.4.6'
 # Decorate logic to keep it of the views and helper methods
 gem 'draper'
 
-# http client
-gem 'httparty'
+# Keep-Alive a http connection
+gem 'net-http-persistent'
```

Comment:
```suggestion
gem 'net-http-persistent'

# HTTP client library
gem 'faraday'
```

Worth specifying this explicitly as a dependency so its clear that this is the http lib of choice for this project (so we don't see the reintroduction of httparty or other).